packages/config/src/tokens/tokens.jsonc

```diff
@@ -584,6 +584,10 @@
       "symbol": "PNT",
       "address": "0x89Ab32156e46F46D02ade3FEcbe5Fc4243B9AAeD"
     },
+    {
```

Comment:
@antooni we need a test that informs the developer that they need to run a script!